var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -23,6 +23,7 @@ class Gromacs(CMakePackage):
     maintainers = ['junghans', 'marvinbernhardt']
 
     version('master', branch='master')
+    version('2021-rc1', sha256='baab9f9c7a659f0777f0ff06866e88685a4b06d22c0f431f5688a9a559f0a1e1')
     version('2020.4', sha256='5519690321b5500c7951aaf53ff624042c3edd1a5f5d6dd1f2d802a3ecdbf4e6')
```

Comment:
You should add `preferred=True` to this version so that a release candidate isn't installed by default.

Reply:
Good catch. Updated preference to 2020.4.